onnxruntime/core/platform/windows/logging/etw_sink.cc

```diff
@@ -137,28 +137,47 @@ void NTAPI EtwRegistrationManager::ORT_TL_EtwEnableCallback(
 EtwRegistrationManager::~EtwRegistrationManager() {
   std::lock_guard<OrtMutex> lock(callbacks_mutex_);
   callbacks_.clear();
-  ::TraceLoggingUnregister(etw_provider_handle);
+  if (initialized_ || initializing_) {
+    std::lock_guard<OrtMutex> init_lock(init_mutex_);
+      assert(!initializing_);
```

Comment:
The linter is failing because of the spacing here: https://github.com/microsoft/onnxruntime/actions/runs/10167712594/job/28120697461?pr=21566#step:6:40

Reply:
As a human linter, that immediately stood out to my eyes too :b.